spec/support/shared/hashing_shared_context.rb

```diff
@@ -25,5 +25,7 @@
     Doorkeeper.configure do
       hash_application_secrets
     end
+
+    ::Doorkeeper::Application.set_bcrypt_hashing_if_available
```

Comment:
Why do we need to manually call it in the specs?

Reply:
The include happens one time when the model is loaded, which with random order will happen anywhere in the specs. If I stub the availibility of bcrypt in the one spec and would not reset it in the next one, the spec will end up running the non-bcrypt configured application model.

Reply:
Ahh, OK, I got it. That's OK